docs/topics/mta-web-config-jira-connection.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-connection_{context}"]
+= Configuring Jira connection
+
+To be able to create Jira issues from within the {ProductName} ({ProductShortName}) {WebName} and to track application migrations, you need to create a Jira instance in {ProductShortName} and establish a connection to that instance.
+
+.Procedure
+
+. In *Administration* view, under *Issue Management*, click *Jira*.
++
+The *Jira configuration* page opens.
+. Click *Create new*.
++
+The *New instance* window opens.
+. Enter the following information:
+* Name of the instance
+* URL of the web interface of your Jira account
+* Instance type - select either *Jira Cloud* or *Jira Server/Datacenter* from the drop-down list
+* Credentials - select from the drop-down list
++
```

Comment:
Suggestion: remove "drop-down" in lines 22 and 23. I prefer colons instead of hyphens for descriptions, but as long as the section is consistent, I leave that to you.